torch/_refs/special/__init__.py

```diff
@@ -93,6 +119,27 @@ def multigammaln(a: TensorLikeType, p: int) -> TensorLikeType:
     return torch.sum(torch.lgamma(a.unsqueeze(-1) + b), dim=-1) + c
 
 
+@register_decomposition(torch.ops.aten.special_ndtr)
+@out_wrapper()
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("a",),
+    type_promotion_kind=utils.ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT,
+)
+def ndtr(a: TensorLikeType) -> TensorLikeType:
+    a_sqrt_2 = a / 1.414213562373095048801688724209698
```

Comment:
This doesn't look right -- shouldn't this expression be `a * M_SQRT1_2 `? 

https://github.com/pytorch/pytorch/blob/2a88f1b2d86a1fdc7380db768e67b18c24d199c4/aten/src/ATen/native/UnaryOps.cpp#L570

Reply:
Hi, @mruberry! Thanks for the question. I believe the formula is right:

Here, it's `a * M_SQRT1_2`, where M_SQRT1_2 is `1/√2` = `0.70710678118654752440`.

And in my impl, I had written `a / √2.`

Reply:
That makes sense but maybe the multiplication is preferable? Multiplication is typically faster than division

Reply:
I appreciate your input! I've already updated using multiplication :p Thank you :)